scripts/ccpp_prebuild_config_FV3.py

```diff
@@ -30,8 +30,6 @@
 #    'ccpp/physics/physics/module_nst_model.f90',
 #    'ccpp/physics/physics/module_nst_parameters.f90',
 #    'ccpp/physics/physics/module_nst_water_prop.f90',
-    'ccpp/physics/physics/module_mp_thompson_hrrr_radar.F90',
```

Comment:
These references to Thompson should not be removed, both physics can be compiled into the library (however, they were removed by a PR that is not yet included in your branch).